help_contextual_popup/README.rst

```diff
@@ -0,0 +1,35 @@
+Add contextual help popup
```

Comment:
PLZ use the latest MT template as base for your readme file. The licence badge is missing and as well as the bug tracker section. (https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)